src/components/EventCard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import { CardContent, Typography } from '@material-ui/core';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import useStyles from './util/useStyles';
+import CardMedia from '@material-ui/core/CardMedia';
+import stockEvent from '../res/stockEventPhoto.jpg';
+
+const EventCard = ({ title, description, startDate, endDate, expired = false, img = stockEvent }) => {
+	const classes = useStyles();
+	const zIndex = expired ? -1 : 0;
+	let className = 'card-top-element';
+	className += expired ? ' card-top-element-disabled' : '';
+	return (
+		<div className={className}>
+			<Card className={classes.eventCard} style={{ position: 'relative', zIndex: zIndex }}>
+				<CardMedia
+					className={classes.media}
+					image={img}
+					title={title}
+				/>
+				<CardContent>
+					<Typography variant={'h3'}>
+						{title}
+					</Typography>
+					<Typography variant={'h5'} color="textSecondary">
+						{description.substring(0, 50)}
+					</Typography>
+					<Typography variant={'subtitle2'} color="secondary">
```

Comment:
This is more of a stylistic change, but where possible can you replace things like `variant={'subtitle2'}` with `variant="subtitle2"`? This affects the h5 and h3 above